test/e2e/persistent_volumes-vsphere.go

```diff
@@ -99,11 +102,12 @@ var _ = framework.KubeDescribe("PersistentVolumes:vsphere", func() {
 		if c != nil {
 			if clientPod != nil {
 				clientPod, err = c.CoreV1().Pods(ns).Get(clientPod.Name, metav1.GetOptions{})
```

Comment:
don't really neeed this block. It's simpler to just call `DeletePodWithWait` passing in clientPod. Nil pod is handled. Non-existent pod is handled too.